docs/UserGuide.md

```diff
@@ -209,9 +231,9 @@ Do take some time to read the following note to better understand how you can us
   * `addClient n/John Tan p/12345678`
   * `addClient n/John p/12345678 e/John@abc.com b/12122000`
 
-* This symbol `||` indicates that only one of the optional parameters can be used. <br> 
+* This symbol `||` indicates that only one of the optional parameters can be used. <br>
   **Using more than one optional parameter is strictly not allowed.** <br>
-  e.g. `listClient [pd/PRODUCT || b/BIRTHDAY]` can be used as 
+  e.g. `listClient [pd/PRODUCT || b/BIRTHDAY]` can be used as
   * `listClient pd/Product1`
   * `listClient b/week`
```

Comment:
Would it be better to include `but not including listClient pd/Product1 b/week`?

Reply:
This one I dun think under me ah, I just did some formatting I believe. Probably will leave it to the person doing it to settle it.

Reply:
i think jn it was mentioned one of the issues, we need to include even faulty examples https://github.com/AY2223S1-CS2103T-W16-4/tp/issues/265. will this be done?

Reply:
to be added to the section under feature